### PR TITLE
Move Devtools tabs into Implementer Tools

### PR DESCRIPTION
**Pull request by brandones**
https://github.com/openmrs/openmrs-esm-devtools/pull/31

Screenshot from 2020-08-22 22-36-51
Screenshot from 2020-08-22 22-36-59

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@openmrs/esm-implementer-tools",
```
Comment (donaldkibet) [inline package.json]:
```suggestion
  "name": "@openmrs/esm-implementer-tools-app",
```

File: src/root.component.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    </>
+  );
+}
+
+export default Root;
+//  openmrsRootDecorator({
+//   featureName: "Implementer Tools",
+//   moduleName: "@openmrs/esm-implementer-tools-app",
+// })(Root);
+
```
Comment (donaldkibet) [inline src/root.component.tsx]: we can delete this if it's not used

**Review by donaldkibet — APPROVED** [review]
:+1:

File: src/root.component.tsx (hunk 2 above)
Comment (brandones) [reply src/root.component.tsx]: Whoops, commented it out while debugging something and forgot to re-enable it. Thanks!

**Pull request merged**